builds/sbt_image_builder.Dockerfile

```diff
@@ -1,15 +1,9 @@
-FROM pvansia/scala-sbt:0.13.13
+FROM wellcome/sbt_wrapper
```

Comment:
This change means we'll see a small saving in builds as well, as we don't need to install docker twice in a run.